cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1236,11 +1237,18 @@ public void removeNotificationReceivedListener() {
     }
 
     public boolean fireNotificationOpenedListener(final NotificationOpenedResult openedResult) {
+        return fireNotificationOpenedListener(openedResult, null);
+    }
+
+    public boolean fireNotificationOpenedListener(final NotificationOpenedResult openedResult, Context context) {
         if (openedResult.getNotification().getAppBanner() != null) {
             showAppBanner(openedResult.getNotification().getAppBanner(), openedResult.getNotification().getId());
         }
 
         if (notificationOpenedListener == null) {
+            if(openedResult.getNotification().getUrl() != null && !openedResult.getNotification().getUrl().equalsIgnoreCase("")){
```

Comment:
Code style (spaces & add new line)

Reply:
Please also consider this:

https://github.com/cleverpush/cleverpush-android-sdk/blob/91f649d927a0c6c7cb2cdd4c36e01b6b4a0ccee8/cleverpush/src/main/java/com/cleverpush/CleverPush.java#L489